src/lib/SvelteNodeViewRenderer.svelte.ts

```diff
@@ -26,6 +26,7 @@ export interface SvelteNodeViewRendererOptions extends NodeViewRendererOptions {
   update: ((props: RendererUpdateProps) => boolean) | null;
   as?: string;
   attrs?: AttrProps;
+  context?: Map<any, any>;
```

Comment:
haven't checkout the code yet. shouldn't this be string, unknown?

Reply:
Mentioned just for the sake of eslint. planning to import some shared configuration where any is explicitly avoided. might have to exclude this line, hence said

Reply:
Maybe `ReturnType<typeof getAllContexts>` could be used to not use any `any`.